src/Response/CSVResponse.php

```diff
@@ -0,0 +1,121 @@
+<?php declare(strict_types = 1);
+
+namespace Contributte\Application\Response;
+
+use Nette\Application\IResponse;
+use Nette\Http\IRequest as IHttpRequest;
+use Nette\Http\IResponse as IHttpResponse;
+
+/**
+ * CSV file download response
+ *
+ * @author Pavel Janda <me@paveljanda.com>
+ */
+class CSVResponse implements IResponse
+{
+
+	/** @var string */
```

Comment:
Personally, I don't like these short docblocks. In all project we use:

```
/**
 * @var string
 */
protected $delimiter;
```

And so does Symfony, Guzzle, etc.

Reply:
We agreed with @paveljanda to write it both ways. We'll see in the future.